ci/benchmark/build.sh

```diff
@@ -82,8 +82,8 @@ if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
     gpuci_logger "gpuci_mamba_retry update dask"
     gpuci_mamba_retry update dask
 else
-    gpuci_logger "gpuci_mamba_retry install conda-forge::dask==2022.03.0 conda-forge::distributed==2022.03.0 conda-forge::dask-core==2022.03.0 --force-reinstall"
-    gpuci_mamba_retry install conda-forge::dask==2022.03.0 conda-forge::distributed==2022.03.0 conda-forge::dask-core==2022.03.0 --force-reinstall
+    gpuci_logger "gpuci_mamba_retry install conda-forge::dask>=2022.03.0 conda-forge::distributed>=2022.03.0 conda-forge::dask-core>=2022.03.0 --force-reinstall"
+    gpuci_mamba_retry install conda-forge::dask>=2022.03.0 conda-forge::distributed>=2022.03.0 conda-forge::dask-core>=2022.03.0 --force-reinstall
```

Comment:
I've been wondering if a less cumbersome way of achieving this would be to manually remove `dask/label/dev` from our list of channels here? Maybe something like

```suggestion
    gpuci_logger "gpuci_mamba_retry install dask>=2022.03.0 --force-reinstall"
    gpuc_conda_retry config --remove channels dask/label/dev
    gpuci_mamba_retry install dask>=2022.03.0 --force-reinstall
```

Not sure if removing the channel would persist in other runs though, cc @ajschmidt8 @jakirkham if this solution makes sense

Reply:
I remember trying something similar previously but didn't seem to work, testing it now again in CI

Reply:
Yeah agree. Recall Prem tried several things, but don't recall if this one (and if so whether we encountered any issues with that)

Reply:
If this doesn't work, happy to merge in as is since the current setup is working

Reply:
Will this actually install Dask main or just the latest release?

Reply:
We use nightlies from the Dask channel in some cases. Depends if the channel `dask/label/dev` is used

Reply:
And do nightlies really mean "once every night", or does it mean it's a new package after every merge?

Reply:
Every merge. They are equivalent to using `main`, but also include Dask's dependencies

Reply:
Sounds good, thanks for confirming.

Reply:
@charlesbluca It didn't work the way we would expect it to just like last time, log here: https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci-22-06/job/cudf/job/prb/job/cudf-gpu-test/CUDA=11.0,GPU_LABEL=driver-450,LINUX_VER=centos7,PYTHON=3.8/150/consoleText 
hence reverting these temp changes.